src/cryptenroll/cryptenroll.c

```diff
@@ -622,6 +626,10 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_LIST_DEVICES:
+                        blockdev_list(BLOCKDEV_LIST_SHOW_SYMLINKS|BLOCKDEV_LIST_REQUIRE_LUKS);
+                        return 0;
```

Comment:
Hmm, I don't think we should eat up errors here? blockdev_list already handle some errors gracefully, and OOM/device matching errors are not something we should ignore?